### ISSUE TITLE
"Numerical issues" Warning building model, then very bad accuracy

### ISSUE THREAD

**Issue by dbuezas** [auduno/clmtools#1]
I added some new points to the face training images and configured the tool to take them into account.
I get the following Wargning:

```
/Library/Python/2.7/site-packages/sklearn/preprocessing/data.py:153: UserWarning: Numerical issues were encountered when centering the data and might not be solved. Dataset may contain too large values. You may need to prescale your features.
  warnings.warn("Numerical issues were encountered "
/Library/Python/2.7/site-packages/sklearn/preprocessing/data.py:169: UserWarning: Numerical issues were encountered when scaling the data and might not be solved. The standard deviation of the data is probably very close to 0.
```

And the precision of the tracker is horrible now.

Any tips? I started with the provided anotations.csv and adapted the model_viewer and pdm_builder to account for the new points.

**Comment by dbuezas** [comment]
Update: I generated the model.js from the original images and annotations and the result is also bad. Is there any trick I could try?

**Comment by auduno** [comment]
Hi, sorry for not replying to this sooner. I'm very busy finishing my master thesis, and also looking after a newborn, so don't have much time to follow up issues. Did you figure this out? As I understand you're trying to add more points to the model than the default 71.

**Comment by dbuezas** [comment]
Hi Audun,
thanks for your message and I wish you luck with your master thesis.

Yes that's right, that is what I was trying to do. I also tried just to
generate the model with the provided training data and somehow the results
were also bad.

I am not working in this company anymore but I think they still want to get
it going, feel free to contact them, it might be a good freelancing
opportunity, I added the contact person as cc.

Best regards,
David

David Buezas
